### ISSUE TITLE
add fountain.io support for screanplay/script writing

### ISSUE THREAD

**Issue by fenpaws** [BookStackApp/BookStack#2563]
**Describe the feature you'd like**
It would be awesome to add [fountain.io](https://fountain.io/) support, this allows writers who write their movie/screenplay scripts inside of bookstack, to be rendered correctly. 

**Describe the benefits this feature would bring to BookStack users**
For example, my friends and I use bookstack to share our scripts for movie projects but with traditional markdown, it's hard to follow the industry guidelines.  This would greatly enhance the visual clarity to everyone who needs a clear structure.

**Comment by ssddanbrown** [comment]
Thanks for the request @fenpaws,

I'd consider support for this to be out of scope for the main project, due to the limited audience it'd be focused upon.
That said, maybe I can find a workaround for you.

What do you see fountain.io support looking like within BookStack? What would BookStack need to do?

**Comment by ssddanbrown** [comment]
Closing since there has been no follow-up.
